src/apps/backend/modules/account/internal/account_reader.py

```diff
@@ -35,15 +35,15 @@ def get_account_by_username_and_password(
     return account
   
   @staticmethod
-  def get_account_by_id(*, params: AccountSearchByIdParams) -> AccountModel:
+  def get_account_by_id(*, params: AccountSearchByIdParams) -> Account:
     account = AccountRepository.account_db.find_one({
       "_id": ObjectId(params.id),
       "active": True,
     })
     if account is None:
       raise AccountNotFoundError(f"Account with id:: {params.id}, not found")
 
-    return AccountModel(**account)
+    return AccountUtil.convert_account_model_to_account(AccountModel(**account))
```

Comment:
What's the difference between `Account ` and `AccountModel` ?

Reply:
`Account` is the structure of user account module while `AccountModel` handles account storage and interation with DB
In short `AccountModel` is just like how we use `AccountDB` in MERN boilerplate